packages/core/src/stateUtils.ts

```diff
@@ -1548,35 +1591,23 @@ function resolveAndExecuteActionsWithContext(
           : undefined;
 
     function executeAction() {
```

Comment:
what about actions such as `sendTo`? how those are supposed to be handled with `transition`? they are not passed through this `executeAction` function

Reply:
Oh, good call. There are some built-in actions such as `assign(…)` and `raise(…)` that immediately affect the next `state`, but `sendTo(…)` may affect it later if it's a delayed send.

Do you think we should include built-in actions anyway in `actions`? What should they do if the `state` already represents the action taken by them?

Reply:
This is done - `sendTo`, `raise`, `cancel` built-in actions are now handled through `executeAction` as well